### DISCUSSION TITLE
How to solve this water meter reading problem.

### DISCUSSION THREAD

**Discussion by haarkop** [jomjol/AI-on-the-edge-device#2918] [General]
Hello, I have a problem and do not know how to deal with it. In my case, 8 digits are responsible for the value including 3 digits are numbers after the decimal point. The result should be 1227.34979 m3, but it reads 1277349.79. The problem is in the decimal point.

![image](https://github.com/jomjol/AI-on-the-edge-device/assets/85073788/754c71ba-b68b-4cd5-b0b5-8c0985e2c08f)

**Comment by caco3** [comment]
You have to configure the Decimal shift correctly, see https://jomjol.github.io/AI-on-the-edge-device-docs/Parameters/#parameter-numberdecimalshift
